plugin/CactbotEventSource/CactbotEventSource.csproj

```diff
@@ -52,28 +52,28 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Advanced Combat Tracker">
-      <HintPath>..\ThirdParty\ACT\Advanced Combat Tracker.exe</HintPath>
+      <HintPath>E:\Advanced Combat Tracker\Advanced Combat Tracker.exe</HintPath>
```

Comment:
And as Akurosia says, you should revert all the changes to csproj files to fix the build errors.